atr/templates/tutorial.html

```diff
@@ -55,7 +55,7 @@ <h2 id="compose">Compose</h2>
          alt="Illustration: Review the results of your checks, and use the Start voting button." />
   </p>
   <p>
-    A form to start the vote is displayed. If necessary, use its ability to preview the vote announcement email body. When you submit the form, ATR sends an announcement message in your name to the chosen mailing list.
+    A form to start the vote appears. If necessary, use its ability to preview the vote announcement email body. When you submit the form, ATR sends an announcement message in your name to the chosen mailing list.
```

Comment:
I originally wrote "appears", but it sounded strange to me and I struggled to find a replacement. I would certainly accept reverting it to "appears", but alternative suggestions to choose from would be welcome.